build.proj

```diff
@@ -80,8 +80,9 @@
 
   <Import Project="build/Microsoft.DotNet.Cli.Prepare.targets" />
   <Import Project="build/Microsoft.DotNet.Cli.Compile.targets" />
+  <Import Project="build/Microsoft.DotNet.Cli.Signing.targets" />
   <Import Project="build/Microsoft.DotNet.Cli.Package.targets" />
-  <Import Project="build/Microsoft.DotNet.Cli.Test.targets" />
+  <Import Project="build/Microsoft.DotNet.Cli.Test.targets" Condition="'$(SkipMsBuild15Projects)' != 'True'" />
```

Comment:
MicroBuild does not support the .NET Core MSBuild, so the full-framework MSBuild is used to build the signing targets. However, the new build machines do not have MSBuild 15, so they use 14. This means that projects which use the new MSBuild 15 syntax fail to be imported.
They're not needed for the signing, so their importing is skipped by using a new `SkipMsBuild15Projects` property which is set to true in the signing build steps.